docs/node-configuration.md

```diff
@@ -16,6 +16,7 @@ node-related settings described in the table below.
 
 | Section | Type | Default value | Description |
 | --- | --- | --- | --- |
+| ArchivalNodesSync | `bool` | `false` | Determines whether the server is asking for old blocks (remote from the current height for more than `MaxTraceableBlocks` blocks) from archival nodes only if syncing via P2P. |
```

Comment:
I think it should marked as temporary till the feature is properly added to N3 protocol.